pkg/controller/usersignup/usersignup_controller.go

```diff
@@ -242,9 +243,11 @@ func (r *ReconcileUserSignup) Reconcile(request reconcile.Request) (reconcile.Re
 			}
 		} else {
 			// If there isn't an email-hash label, then the state is invalid
+			log.Info("missing label on usersignup", "label", toolchainv1alpha1.UserSignupUserEmailHashLabelKey, "username", instance.Spec.Username)
 			return reconcile.Result{}, r.updateStatus(reqLogger, instance, r.setStatusMissingEmailHash)
 		}
 	} else {
+		log.Info("missing annotation on usersignup", "annotation", toolchainv1alpha1.UserSignupUserEmailAnnotationKey, "username", instance.Spec.Username)
```

Comment:
```suggestion
		reqLogger.Info("missing annotation on usersignup", "annotation", toolchainv1alpha1.UserSignupUserEmailAnnotationKey)
```

Reply:
same question here: what's wrong with printing the username?

Reply:
It's already have been added:
```
reqLogger = reqLogger.WithValues("username", instance.Spec.Username)
```

Reply:
🤦‍♂

Reply:
done in 1c0e3c3